shells/web-shell-2/index.html

```diff
@@ -0,0 +1,109 @@
+<!doctype html>
```

Comment:
What is web-shell-2?

Reply:
I made that folder to explore login-flows, I'll remove it.